### DISCUSSION TITLE
The document link on the official website does not jump to the current latest

### DISCUSSION THREAD

**Discussion by ZhangChengLin** [ColorlibHQ/AdminLTE#3690] [General]
https://adminlte.io/docs/

```
view-source:https://adminlte.io/docs/
```

At present, the document on the official website is the document that jumped to 2.x. This is an outdated document address, which makes people feel that it has not been maintained for a long time and cannot find a new document address. In fact, it has been actively maintained.

---

other:
You should not rely solely on viewing the source code to learn the latest, improve the document, lower the threshold of use, add code running examples of individual components, and also determine the effectiveness of the document to a certain extent. Add the source address of the document page to the document, maybe Sometimes it works.

**Comment by ZhangChengLin** [comment]
@danny007in

**Reply by danny007in** [reply]
ask to @REJack 

I am just collaborator

**Reply by ZhangChengLin** [reply]
> ask to @REJack
> 
> I am just collaborator

Oh come on

**Reply by REJack** [reply]
The fact is that the old page is built from the original author of the template, since the move to ColorlibHQ we had no login infos or other infos about the page since the old author has created the page by his own knowledge. I work on a new page but at the moment I hustle between my jobs and my OSS projects

**Reply by ZhangChengLin** [reply]
> The fact is that the old page is built from the original author of the template, since the move to ColorlibHQ we had no login infos or other infos about the page since the old author has created the page by his own knowledge. I work on a new page but at the moment I hustle between my jobs and my OSS projects

They should be adjusted to facilitate development, and it doesn't take too much time, right?
